app/helpers/activity_helper.rb

```diff
@@ -12,7 +12,7 @@ def step_is_complete_or_next?(activity:, step:)
   end
 
   def activity_back_path(current_user:, activity:)
-    if activity.programme? && current_user.service_owner?
+    if activity.parent.present? && activity.programme? && current_user.service_owner?
```

Comment:
We have agreed to lose these back links so this change is a very quick plaster to make it just work for now.